test/remote-connector.test.js

```diff
@@ -87,3 +91,55 @@ describe('RemoteConnector', function() {
     });
   });
 });
+
+describe('Custom Path', function() {
+
+  var test = this;
+
+  before(function(done) {
+    
+    var db_config = {
+      connector: loopback.Memory,
+      name: 'db'
+    };
+    var db = loopback.createDataSource('db', db_config);
+
+    var ServerModel_config = {
+      dataSource: 'db',
+      http: {
+        path: '/custom'
+      }
+    };
+
+    var ServerModel = loopback.PersistedModel.extend('TestModel', {foo: 'string'}, ServerModel_config);
+    ServerModel.attachTo(db);
+
+    server = test.server = createAppWithRest();
+    server.model(ServerModel);
+    server.dataSource(db, db_config);
```

Comment:
@bajtos 
> This setup is a bit convoluted

The "convoluted" setup follows the same pattern as the loopback-boot module. Your simplified proposal does not correctly attach `http.path` and the test fails.

> BTW I personally tend to inline *_config variables in the place where they are used, especially when they are used only once.

I usually do too except in this case the lines were quite long. I tried to follow the conventions of the code and config files generated by `slc:loopback`. There appears to be a couple redundancies, but that is a different issue all together.